gulpfile.js

```diff
@@ -252,7 +252,8 @@ gulp.task('ext:test', (done) => {
         workspace = process.cwd();
     }
     process.env.JUNIT_REPORT_PATH = workspace + '/test-reports/ext_xunit.xml';
-    cproc.exec(`code --extensionDevelopmentPath="${workspace}" --extensionTestsPath="${workspace}/out/test" --verbose`,
```

Comment:
we should adopt vscode's testing lib, `vscode-test` npm module

Reply:
will add this in a separate PR, just re-enabling the tests for now